go.mod

```diff
@@ -86,9 +86,9 @@ require (
 	github.com/google/uuid v1.1.2
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2
-	github.com/k3s-io/helm-controller v0.8.3
-	github.com/k3s-io/kine v0.6.0
-	github.com/klauspost/compress v1.11.7
+	github.com/k3s-io/helm-controller v0.9.1
+	github.com/k3s-io/kine v0.6.1
```

Comment:
Need to pull in https://github.com/k3s-io/k3s/commit/246b378a2743e659a2343b60123ba51a38dcc0ac for kine delete fix